programs/programs/urgent_needs/il/il_rent_asst.py

```
@@ -0,0 +1,27 @@
+from ..base import UrgentNeedFunction
+from integrations.clients.hud_income_limits import hud_client
+from integrations.clients.hud_income_limits.client import Section8AmiPercent
+
+
+class IlRenterAssistance(UrgentNeedFunction):
+    ami_percent: Section8AmiPercent = "80%"
+    dependencies = ["income_amount", "income_frequency", "household_size", "county"]
```

Ruff (0.14.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)